package Hello;
message LexiconCast {
  required string v1 = 1;
  required int32 v2 = 2;
}

message IncRequest {
  required int32 v = 1;
  required int32 step = 2;
}

message IncResponse {
  required int32 v = 1;
}

message EchoRequest {
  required string question = 1;
}

message EchoResponse {
  required int32 echoed = 1;
  required string text = 2;
}

message EchoRequest2 {
  required string question = 1;
}

message EchoResponse2 {
  required int32 echoed = 1;
  required string text = 2;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}

service EchoService2 {
  rpc Echo1(EchoRequest) returns (EchoResponse);
  rpc Echo2(EchoRequest2) returns (EchoResponse2);
}
